ccmlib/utils/sni_proxy.py

```diff
@@ -96,26 +104,32 @@ def configure_sni_proxy(conf_dir, nodes_info, listen_port=443):
         """)
     tables = ""
     mapping = {}
-    address, port, host_id, _ = list(nodes_info)[0]
+    address, port, host_id, data_center = list(nodes_info_per_dc)[0]
     tables += f"  ^cql.cluster-id.scylla.com$ {address}:{port}\n"
     mapping['FIRST_ADDRESS'] = address
     mapping['listen_port'] = listen_port
 
-    for address, port, host_id, _ in nodes_info:
+    for address, port, host_id, _ in nodes_info_per_dc:
         tables += f"  ^{host_id}.cql.cluster-id.scylla.com$ {address}:{port}\n"
 
     tmpl = string.Template(sniproxy_conf_tmpl)
-    sniproxy_conf_path = os.path.join(conf_dir, 'sniproxy.conf')
+
+    sni_proxy_path = conf_dir + '/sni_proxy'
+    if not os.path.exists(sni_proxy_path):
+        os.mkdir(sni_proxy_path)
+
+    sni_proxy_for_dc_filename = data_center + '_' + 'sniproxy.conf'
+    sniproxy_conf_path = os.path.join(sni_proxy_path, sni_proxy_for_dc_filename)
 
     with open(sniproxy_conf_path, 'w') as fp:
         fp.write(tmpl.substitute(TABLES=tables, **mapping))
 
     return sniproxy_conf_path
 
 
-def start_sni_proxy(conf_dir, nodes_info, listen_port=443):
-    address, _, _, _ = list(nodes_info)[0]
-    sniproxy_conf_path = configure_sni_proxy(conf_dir, nodes_info, listen_port=listen_port)
+def start_sni_proxy(conf_dir, nodes_info_per_dc, listen_port=443):
```

Comment:
please don't change the APIs, this is breaking the python-driver code tests.
this are calling this API directly

Reply:
the python-driver are not using ccm from  commandline like driver tests

Reply:
I have reverted API changing modifications.